Makefile

```diff
@@ -9,22 +9,23 @@ GOBIN?=$(GOPATH)/bin
 # Build parameters
 OUT?=./out
 DOCKER_TMP?=$(OUT)/docker_temp/
-DOCKER_TAG?=aeraki/consul2istio:latest
+DOCKER_TAG?=docker.io/tanjunchen/consul2istio:latest
 BINARY_NAME?=$(OUT)/consul2istio
 BINARY_NAME_DARWIN?=$(BINARY_NAME)-darwin
 MAIN_PATH_CONSUL_MCP=./cmd/consul2istio/main.go
 
 build:
-	CGO_ENABLED=0 GOOS=linux  $(GOBUILD) -o $(BINARY_NAME) $(MAIN_PATH_CONSUL_MCP)
+	CGO_ENABLED=0 GOARCH=amd64 GOOS=linux  $(GOBUILD) -o $(BINARY_NAME) $(MAIN_PATH_CONSUL_MCP)
```

Comment:
the default goarch is supposed to be x86

Reply:
I make this parameter configurable.